sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkLauncher.java

```diff
@@ -302,7 +302,7 @@ private NexmarkPerf currentPerf(
       effectiveEnd = eventEnd;
     }
 
-    if (effectiveEnd >= 0 && eventStart >= 0 && effectiveEnd >= eventStart) {
+    if (eventStart >= 0 && effectiveEnd >= eventStart) {
```

Comment:
true that this expression is equivalent and simpler but less readable. I would prefer a revert